docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +107,261 @@ To manage file storages:
    tizen.filesystem.removeStorageStateChangeListener(watchID);
    ```
 
-## Creating and Deleting Files and Directories
-
-You can create files and directories using the `createFile()` and `createDirectory()` methods. The file or directory is created relative to the current directory that the operation is performed on.
-
-> **Note**
-> Do not use "." or ".." characters in the directory or file path components.
-
-To create and delete files and directories:
-
-1. To create a file in the current directory, use the `createFile()` method of the `File` interface (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/filesystem.html#File), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/filesystem.html#File), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/filesystem.html#File) applications):
-
-   ```
-   var documentsDir, newFile;
-   tizen.filesystem.resolve('documents', function(result) {
-       documentsDir = result;
-       newFile = documentsDir.createFile('newFilePath');
-   });
-   ```
-
-2. To create a directory within the file system, use the `createDirectory()` method.
+## Creating and Deleting Directories
 
-   The directory (and any sub-directories defined in the method parameter) is created relative to the current directory where the operation is performed on.
+You can create directories using the `createDirectory()` method. The directory is created at `path` specified in parameter.
+You can delete directories using  `deleteDirectory()` method. Target pointed by `path` specified will be deleted from filesystem.
 
-   ```
-   var newDir = documentsDir.createDirectory('newDir');
-   var anotherNewDir = documentsDir.createDirectory('newDir1/subNewDir1');
-   ```
+1. To create a directory within the file system, use the `createDirectory()` method.
```

Comment:
Please change the ordered list under "Creating and Deleting Directories" to unordered list (bullets) as it is not a sequential or a procedural step.